pypeit/core/flat.py

```diff
@@ -1,488 +1,491 @@
-""" Core module for methods related to flat fielding
+"""
+Core module for methods related to flat fielding.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../links.rst
 """
 import inspect
+import copy
+import os
 
 import numpy as np
-import os
+from scipy import interpolate, ndimage
+from matplotlib import pyplot as plt
+
+from IPython import embed
 
-from scipy import interpolate
 from pypeit import msgs
 from pypeit.core import parse
 from pypeit.core import pixels
 from pypeit.core import tracewave
-from scipy.interpolate import interp1d
-
-from pypeit import debugger
 from pypeit import utils
 from pypeit.core import pydl
-from matplotlib import pyplot as plt
-import copy
-from IPython import embed
-
-import scipy
 
-def tweak_slit_edges(slit_left_in, slit_righ_in, ximg_fit, normimg, tweak_slits_thresh,
-                     tweak_slits_maxfrac):
-    """
-    DOC THIS!
+# TODO: Put this in utils
```

Comment:
I realize that you have reworked these codes including changing naming etc. But please do not delete the originals, instead move them to deprecated. You can get the originals out of the current develop branch.

Reply:
Okay, will do.